pygmt/src/grdproject.py

```diff
@@ -0,0 +1,86 @@
+"""
+grdproject - Forward and inverse map transformation of grids.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    J="projection",
+    I="inverse",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(R="sequence")
+def grdproject(grid, **kwargs):
+    r"""
+    Change projection of gridded data between geographical and rectangular.
+
+    This module will project a geographical gridded data set onto a
+    rectangular grid. If ``inverse`` is ``True``, it will project a
+    rectangular coordinate system to a geographic system. To obtain the value
+    at each new node, its location is inversely projected back onto the input
+    grid after which a value is interpolated between the surrounding input
+    grid values. By default bi-cubic interpolation is used. Aliasing is
+    avoided by also forward projecting the input grid nodes. If two or more
+    nodes are projected onto the same new node, their average will dominate in
+    the calculation of the new node value. The ``region`` parameter can be
+    used to select a map region larger or smaller than that implied by the
+    extent of the grid file.
```

Comment:
Description is missing 3 sentences from https://github.com/GenericMappingTools/gmt/blame/6.2.0/doc/rst/source/grdproject.rst#L40-L44.